model/config.go

```diff
@@ -33,6 +33,7 @@ type CedarConfig struct {
 	Flags          OperationalFlags          `bson:"flags" json:"flags" yaml:"flags"`
 	Service        ServiceConfig             `bson:"service" json:"service" yaml:"service"`
 	ChangeDetector ChangeDetectorConfig      `bson:"change_detector" json:"change_detector" yaml:"change_detector"`
+	ProxyService   ProxyServiceConfig        `bson:"proxy_service" json:"proxy_service" yaml:"proxy_service"`
```

Comment:
Can we make this more descriptive? `ProxyService` is too general, it should indicate that it is for signal processing analytics. I would also recommend nesting the `ProxyServiceConfig` struct in the `ChangeDetectorConfig`.